gems/pending/util/system/evm_watchdog.rb

```diff
@@ -1,7 +1,11 @@
 require 'time'
+require 'linux_admin'
+require 'fileutils'
 
 module EvmWatchdog
   PID_LOCATION = '/var/www/miq/vmdb/tmp/pids/evm.pid'.freeze
+  UPDATE_FILE  = '/var/www/miq/vmdb/tmp/miq_update'.freeze
```

Comment:
Wow, what I meant to say was that it would be nice if the caller called new with a path so this class /module doesn't need to know the appliance directories. Future....